### PR TITLE
Test

### PR DESCRIPTION
**Pull request by aditansh**

### CODE REVIEW

**Comment by aditansh** [comment]
@anandrajaram21 check this

File: frontend/src/pages/signup.tsx [hunk 1]
```diff
@@ -403,18 +422,28 @@ export default function Home() {
...
                           : ""
                       }`}
                     >
                       <option disabled value="" label="Choose your gender" />
-                      <option value="Male" label="Male" />
-                      <option value="Female" label="Female" />
+                      <option
+                        value="Male"
+                        label="Male"
+                        className="text-black"
```
Comment (sahas-01) [inline frontend/src/pages/signup.tsx:435]: Change to the gray-900 used everywhere

File: frontend/src/pages/signup.tsx [hunk 2]
```diff
@@ -403,18 +422,28 @@ export default function Home() {
...
+                      />
+                      <option
+                        value="Female"
+                        label="Female"
+                        className="text-black"
```
Comment (sahas-01) [inline frontend/src/pages/signup.tsx:440]: Change to the gray-900 used everywhere else

File: frontend/src/pages/signup.tsx [hunk 3]
```diff
@@ -403,18 +422,28 @@ export default function Home() {
...
+                      />
                       <option
                         value="Prefer Not to Say"
                         label="Prefer Not to Say"
+                        className="text-black"
```
Comment (sahas-01) [inline frontend/src/pages/signup.tsx:445]: Change to the gray-900 used everywhere else

File: frontend/src/pages/signup.tsx [hunk 4]
```diff
@@ -575,14 +615,54 @@ export default function Home() {
...
-                      <option value="offline" label="Offline" />
+                      <option
+                        value="online"
+                        label="Online"
+                        className="text-black"
+                      />
+                      <option
+                        value="offline"
+                        label="Offline"
+                        className="text-black"
```
Comment (sahas-01) [inline frontend/src/pages/signup.tsx:625]: Change to the gray used everywhere else - gray-900

File: frontend/src/pages/forms/ProjectForm.tsx [hunk 5]
```diff
@@ -503,7 +508,8 @@ const ProjectForm = () => {
                   Video Link*
                 </label>
                 <p className="mt-2 pb-1 text-sm text-gray-300">
-                  Share a short video demonstarting your project.
+                  Share a short video demonstarting your project (Enter NA if
```
Comment (sahas-01) [inline frontend/src/pages/forms/ProjectForm.tsx]: demonstrating*

File: frontend/src/components/submission.tsx [hunk 6]
```diff
@@ -1,42 +1,123 @@
...
-              type="button"
-              className="text-md absolute bottom-0 right-0 m-4 w-[40vw] rounded-md bg-[#37ABBC]  px-3 py-3 font-semibold text-white shadow-sm transition-all hover:bg-[#288391] focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600 md:mx-8 md:w-64 lg:mx-8 lg:w-64"
-              onClick={() => {
-                router.push("/forms/IdeaForm");
-              }}
-            >
-              Add a submision
-            </button>
-          </div>
+            </p> */}
```
Comment (sahas-01) [inline frontend/src/components/submission.tsx]: Remove comments

**Comment by sahas-01** [comment]
Check comments @aditansh

File: frontend/src/pages/forms/ProjectForm.tsx (hunk 5 above)
Comment (aditansh) [reply frontend/src/pages/forms/ProjectForm.tsx]: done

File: frontend/src/components/submission.tsx (hunk 6 above)
Comment (aditansh) [reply frontend/src/components/submission.tsx]: done

**Pull request merged**